MIPs/mip-mc-request.md

```diff
@@ -0,0 +1,156 @@
+    "method": "provider_request",
+    "params": {
+        "scope": "eip155:59144",
+        "request": {
+            "method": "eth_getBalance",
+            "params": ["0xabc...", "latest"]
+      }
+    }
+    "id": 2
+  }
+]);
+```
+
+In this example, the `provider_request` wraps each JSON-RPC `eth_getBalance` request with one scoped to Ethereum Mainnet (`eip155:1`) and another to Linea (`eip155:59144`).
+
+For consistency and excepting [provider_authorization](mip-mc-handshake.md), all MetaMask JSON-RPC requests will be made through a `provider_request` including those that are `wallet` scoped. See [provider_authorization](mip-mc-handshake.md) for information about supported scopes.
+
+#### Reference Spec
+The proposed changes have been specified in a machine-readable format in the following PR against the `MetaMask/api-specs` repo: [add URL here]()
+
+### Error Handling
+If MetaMask is not connected to the chain specified in the `scope`, an error should be returned. The error message should guide the dapp to request the addition of the appropriate network.
+
+If an invalid `scope` is specified, then an error should be returned.
+
+## Backwards Compatibility
+The `provider_request` method is introduced through the [Multichain API](mip-mc-delivery.md), which is completely separate from the existing JSON-RPC methods. Existing JSON-RPC that are called without the `provider_request` wrapper would continue to work as they currently do.
+
+## Test Cases
+General test cases should cover:
+- Successful request with correct `scope`
+- Error handling when `scope` is not provided
+- Error handling when an unsupported `scope` is provided
+
+Test cases for specific JSON-RPC `method` calls are outside the scope of this proposal.
+
+## Caveats
+[List any potential drawbacks, limitations, or risks associated with the proposal]
+
+## Implementation
+MetaMask's JSON-RPC request-handling code needs to be updated to support this new `provider_request` method. Appropriate UI updates are also needed to ensure that users are sufficiently informed and aware of what they are interacting with or being asked to confirm.
+
+A new suite of tests will be added to comprehensively cover key test cases.
+Technical documentation and developer tools will have to be updated to include the Multichain API.
+
+API Maintainers will be implementing the Multichain API in coordination with other MetaMask team members.
+
+## Developer Adoption Considerations
+[Explain any considerations that developers should take into account when adopting this proposal. For example, how will it affect compatibility, and what changes may need to be made to accommodate the proposal?]
+
+The implementation would be designed to be fully backward-compatible. Existing methods for changing networks and executing transactions would remain unaffected. Once there is sufficient industry adoption of the Multichain API, backward-compatibility may be reconsidered and gradually deprecated. 
```

Comment:
It might help to refer specifically to CAIP-25 adoption here-- CAIP-27 is supposed to work even if CAIP-25 isn't adopted (or perfectly implemented), i.e., a "scope" in CAIP-27 *should* correspond to a scopeObject authorized with the handshake, but conceivably some wallets might implement 27 without implementing 25 as a "halfmeasure" or a way of working their way up to the more stateful CAIP-25 concept of scopes by just handling these strings as an interim goal...